truss/templates/server/common/tracing.py

```diff
@@ -11,6 +11,7 @@
 import opentelemetry.sdk.trace as sdk_trace
 import opentelemetry.sdk.trace.export as trace_export
 from opentelemetry import context, trace
+
```

Comment:
I believe https://github.com/basetenlabs/truss/pull/1414 introduced some import changes that `ruff` doesn't like on main - not entirely sure how these got through PR CI.